### PR TITLE
Fixing incorrect telemetry result overwrite

### PR DESCRIPTION
**Pull request by bryceitoc9**
## Description
Fixing telemetry result in SSM doc code.

## Motivation and Context

`telemetry.Result` type was being overwritten by a response object. Unsure how this got past the linter in the first place.

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [ ] A short description of the change has been added to the changelog using the script `npm run newChange`

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by hunterwerlla — APPROVED** [review]

**Review by justinmk3 — APPROVED** [review]

File: src/ssmDocument/commands/createDocumentFromTemplate.ts [hunk 1]
```diff
@@ -66,23 +68,26 @@ export async function createSsmDocumentFromTemplate(): Promise<void> {
...
-
     try {
-        logger.debug(`User selected the ${selection.label} template.`)
-        const textDocument: vscode.TextDocument = await openTextDocumentFromSelection(selection)
-        vscode.window.showTextDocument(textDocument)
+        // User pressed escape and didn't select a template
+        if (selection === undefined) {
+            result = 'Cancelled'
+        } else {
+            logger.debug(`User selected the ${selection.label} template.`)
```
Comment (justinmk3) [inline src/ssmDocument/commands/createDocumentFromTemplate.ts:76]: In general the variable part of a log message should go at the end, where possible. Also don't need punctuation (log messages are not prose).
```suggestion
            logger.debug(`User selected template: ${selection.label}`)
```

**Review by justinmk3 — APPROVED** [review]

**Review by justinmk3 — APPROVED** [review]

**Comment by justinmk3** [comment]
:shipit:  🚀

**Pull request merged**